gddo-server/graph.go

```diff
@@ -41,8 +41,7 @@ func renderGraph(pdoc *doc.Package, pkgs []database.Package, edges [][2]int) ([]
 	p := out.Bytes()
 	if i := bytes.Index(p, []byte("<svg")); i < 0 {
 		return nil, errors.New("<svg not found")
-	} else {
-		p = p[i:]
-	}
+    }
+    p = p[i:]
```

Comment:
Truly weird, as `set list` seemed to show the same line spacing as the lines surrounding it.

Fixed by direct edit on github.